### DISCUSSION TITLE
mutiple beams

### DISCUSSION THREAD

**Discussion by ccmingPipi** [srsran/srsRAN_Project#942] [General Help]
I would like to ask if mutiple beams is supported for SSB next release？？
thank you

**Comment by pgawlowicz** [comment]
Please check our roadmap: https://docs.srsran.com/projects/project/en/latest/general/source/2_features_and_roadmap.html

Multiple Beams will not be supported in the next release.

**Reply by ccmingPipi** [reply]
Will it be realized in the future？

**Reply by pgawlowicz** [reply]
We will probably add this feature eventually. However, we cannot set a timeline for it at this stage.

**Reply by ccmingPipi** [reply]
thank you~
